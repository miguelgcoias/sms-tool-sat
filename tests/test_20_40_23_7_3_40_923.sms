20
6 19 25 6 3 2 3 5 4 2
34 4 42 1 4
3 4 11 4 1 1 1 1
26 12 62 3 3 4 5
4 11 37 4 3 1 3 4
30 6 48 1 6
22 3 25 3 1 1 1
31 23 54 7 4 6 5 2 2 1 3
35 20 95 6 5 6 4 1 2 2
12 18 66 7 3 3 1 3 1 5 2
30 8 54 2 4 4
12 7 19 2 5 2
8 19 65 5 4 6 4 3 2
12 17 63 5 2 5 4 2 4
34 7 48 4 1 2 2 2
35 12 59 1 12
40 10 60 5 2 3 1 2 2
0 8 16 2 4 4
38 21 80 5 2 5 5 6 3
34 19 53 5 2 2 4 6 5
0
0
0
1 2
1 1
0
0
1 14
2 19 10
1 1
0
0
2 1 10
2 13 4
4 1 3 2 5
4 2 3 18 6
1 13
0
0
1 14
