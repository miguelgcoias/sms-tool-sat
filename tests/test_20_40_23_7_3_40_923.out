8
2 34
3 4 5 6 7
4 38 49 53
6 41
7 22 23 24
12 12 17
18 0 8
19 47 58 63 68 74
